tests/pytests/test_config.py

```diff
@@ -859,6 +859,74 @@ def _testImmutableStringConfig(env, configName, ftConfigName, ftDefault,
         _testImmutableStringConfig(env, configName, ftConfigName, ftDefault,
                                    testValue)
 
+
+@skip(cluster=False, redis_less_than='8.1.0')
+def testConfigAPIRunTimeOssGlobalPassword():
+    env = Env(noDefaultModuleArgs=True)
+    if env.env != 'oss-cluster':
+        env.skip()
+
+    env.expect('CONFIG', 'GET', 'search-oss-global-password')\
+        .equal(['search-oss-global-password', 'Password: *******'])
+
+    env.expect('CONFIG', 'SET', 'search-oss-global-password', '123')\
```

Comment:
A comment could make it clearer (fail since it is immutable (only load-time))

Reply:
Comments added for username and password tests.